### PR TITLE
Upgrade the extension's tasks to use Node 20, or fallback to Node 10

### PR DESCRIPTION
**Pull request by hanankem**
- [x] All [tests](https://github.com/jfrog/jfrog-azure-devops-extension#testing) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: tasks/JFrogAudit/task.json [hunk 1]
```diff
@@ -99,9 +99,17 @@
         }
     ],
     "execution": {
+        "Node10": {
+            "target": "audit.js",
+            "workingDirectory": "$(currentDirectory)"
+        },
         "Node16": {
```
Comment (eyalbe4) [inline tasks/JFrogAudit/task.json]: I'm actually not sure Node 16 is needed here.
Perhaps it best to have only Node 10 and Node 20.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: tasks/JFrogAudit/task.json (hunk 1 above)
Comment (hanankem) [reply tasks/JFrogAudit/task.json]: I agree, I will remote Node 16

**Comment by hanankem** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by johnpp143 — COMMENTED** [review]
"Node20_1" may not be required, as azure tasks globally use node 20

**Pull request merged**